yb-voyager/cmd/import.go

```diff
@@ -456,10 +457,33 @@ var onPrimaryKeyConflictActions = []string{
 }
 
 func validateOnPrimaryKeyConflictFlag() {
-	conflictAction := strings.ToUpper(tconf.OnPrimaryKeyConflictAction)
-	if conflictAction != "" {
-		if !slices.Contains(onPrimaryKeyConflictActions, conflictAction) {
+	log.Infof("passed value for --on-primary-key-conflict: %s", tconf.OnPrimaryKeyConflictAction)
+	tconf.OnPrimaryKeyConflictAction = strings.ToUpper(tconf.OnPrimaryKeyConflictAction)
+
+	// ensure that OnPrimaryKeyConflictAction is not changed in case of resumption
+	if !startClean {
+		msr, err := metaDB.GetMigrationStatusRecord()
+		if err != nil {
+			utils.ErrExit("Error getting migration status record: %v", err)
+		}
+
+		if msr != nil && msr.OnPrimaryKeyConflictAction != "" && msr.OnPrimaryKeyConflictAction != tconf.OnPrimaryKeyConflictAction {
```

Comment:
Under what conditions can msr be nil at this point? 
if we do NOT expect it, we should return an error

Reply:
done